test/e2e/yurt/yurt.go

```diff
@@ -54,11 +54,13 @@ func Register() {
 
 		err = ns.DeleteNameSpace(c, YurtE2ENamespaceName)
 		util.ExpectNoError(err)
-
-		ginkgo.By(YurtE2ETestDesc + "yurt_test_create namespace")
-		_, err = ns.CreateNameSpace(c, YurtE2ENamespaceName)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "fail to create namespace")
-
+		util.YurtDescribe(YurtE2ETestDesc+"yurt_test_create namespace", func() {
```

Comment:
Should it be BeforeEach rather than Describe?
We need to create pod in this namespace later in test `pod_operate`.

Reply:
In this manner, we depend on that `yurt_test_create namespace` should run before the `pod_operate`. It is not always the case.